src/Avalonia.Visuals/Media/PathMarkupParser.cs

```diff
@@ -98,6 +99,8 @@ private static Point MirrorControlPoint(Point controlPoint, Point center)
         /// <param name="pathData">The path data.</param>
         public void Parse(string pathData)
         {
+            ThrowIfDisposed();
```

Comment:
Hopefully adding these checks won't slow down path parsing too much?

Reply:
Isn't Path parsed in compile time now?